Source/Public/ZMTransportSession.h

```diff
@@ -130,6 +123,21 @@ extern NSString * const ZMTransportSessionNewRequestAvailableNotification;
 
 @end
 
+@interface ZMTransportSession (AccessToken)
+
+@property (nonatomic, readonly, nullable) ZMAccessToken *accessToken;
+
+/// Sets the access token failure callback. This can be called only before the first request is fired
+- (void)setAccessTokenRenewalFailureHandler:(ZMCompletionHandlerBlock)handler NS_SWIFT_NAME(setAccessTokenRenewalFailureHandler(handler:));
```

Comment:
will the `handler` parameter visible in swift ? 
```suggestion
- (void)setAccessTokenRenewalFailureHandler:(ZMCompletionHandlerBlock)handler NS_SWIFT_NAME(setAccessTokenRenewalFailureHandler(_:));
```
as you have twice handler maybe remove the param or the end of the method